src/modules/integrator-guide/pages/data-privacy/data-privacy-console.adoc

```diff
@@ -2,78 +2,74 @@ include::ROOT:partial$variables.adoc[]
 
 = Data Privacy Console
```

Comment:
Should we also note down the privacy request overview screen and the ability to comment on them, cancel requests, ...